### ISSUE TITLE
gdal3/proj6 fixes

### ISSUE THREAD

**Issue by sckott** [ropensci/geojsonio#159]
from kurt

> Please see the problems shown on
<https://cran.r-project.org/web/checks/check_results_geojsonio.html>.

> Specifically, see the problems for the Debian check systems.

> These are from a recent system upgrade to GDAL 3. 

> Can you please fix your package to work with both GDAL 3 (and PROJ 6) as
well as GDAL 2 (and PROJ 4), as used for the other check systems?

**Comment by sckott** [comment]
we're testing now on GH actions with both older and newer gdal/proj setups, all passing

**Comment by rsbivand** [comment]
Sorry, with released 0.9-0 still seeing:

```
* checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  ══ testthat results  ═══════════════════════════════════════════════════════════
  [ OK: 379 | SKIPPED: 23 | WARNINGS: 0 | FAILED: 15 ]
  1. Error: works with sf (@test-crs_convert.R#26) 
  2. Error: works with sfc (@test-crs_convert.R#35) 
  3. Error: allows supplying a CRS with sf (@test-crs_convert.R#57) 
  4. Error: allows supplying a CRS with sfc (@test-crs_convert.R#62) 
  5. Error: is_wgs84 works with sf (@test-crs_convert.R#67) 
  6. Error: is_wgs84 works with sfc (@test-crs_convert.R#74) 
  7. Failure: geojson_list: convert_wgs84 works with Spatial classes (@test-crs_convert.R#93) 
  8. Failure: geojson_list: convert_wgs84 works with Spatial classes (@test-crs_convert.R#96) 
  9. Failure: geojson_list: convert_wgs84 works with Spatial classes (@test-crs_convert.R#98) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```
as on CRAN. The structure of the **sf** `"crs"` object has changed completely. Maybe drop `test-crs_convert.R` until  things stabilize?
